### DISCUSSION TITLE
How can I hide arguments from the help command?

### DISCUSSION THREAD

**Discussion by murtadapy** [Rapptz/discord.py#8393] [Q&A]
Hi! 

How can I hide arguments from the help command?

```
.command_name

summary...

Arguments:
  name   No description given
  age No description given
```

Thanks for your help in advance!

**Comment by Rapptz** [comment]
You can pass `show_parameter_descriptions=False` to `DefaultHelpCommand`'s `__init__`. For example,

```py
help_cmd = commands.DefaultHelpCommand(show_parameter_descriptions=False)
bot = commands.Bot(intents=intents, command_prefix=commands.when_mentioned, help_command=help_cmd)
```

**Reply by murtadapy** [reply]
I was working around that but didn't know that you can override the default command.

That solved the problem! Thank you very much for everything you've done!
